FaunaDB.Client/Types/Attributes.cs

```diff
@@ -49,13 +49,25 @@ public class FaunaFieldAttribute : Attribute
         /// The default value used for a missing property when decoding an object.
         /// </summary>
         public object DefaultValue { get; set; }
-
+        
         public FaunaFieldAttribute(string name)
         {
             Name = name;
         }
     }
 
+    [AttributeUsage(AttributeTargets.Field | AttributeTargets.Property | AttributeTargets.Parameter)]
+    public class FaunaDate : Attribute
+    {
+
+    }
+
```

Comment:
Cold you please add some doc. and keep the braces in the same line
(see `FaunaIgnoreAttribute` or `FaunaConstructorAttribute`)